playwright_server

```diff
@@ -8,6 +8,7 @@ const fs = require('fs');
 const path = require('path');
 
 // Assume their kit is good
+module.paths.push( '/usr/local/lib/node_modules' );
```

Comment:
There's probably a better way to figure out how to consult "system node" modules.  npm config ls -l seems in the right neighborhood?. npx-import seems like a crazy solution to the problem.

I generally use NVM and that just sets the node lib env correctly.
I'll file a follow up about this.